### DISCUSSION TITLE
Mapsui.Ui.Blazor changing Maps Size

### DISCUSSION THREAD

**Discussion by VengaYBanzai** [Mapsui/Mapsui#2139] [General]
Hi together and thank you for the great project.

I am playing around with the Blazor control (4.0; Windows; VS2022) and cannot figure out, how to overwrite the default height and width of the control. Whatever I do it remains at 300/150. I saw the question in some other discussion, but could not find an answer.

I am probably blind or dump at the end of the day, but help would be appreciated.

Thank you.

**Comment by inforithmics** [comment]
In the Blazor sample I resize the Balzor Control with a div

```
<div class="tab-content">
        <div class="container">
            <div class="row">
                <div class="col border rounded p-2 canvas-container">
                    <MapControlComponent @ref="_mapControl" />
                </div>
            </div>   
        </div>
</div>
```

```
   .tab-content {
        margin-top: 10px;
        border: 1px solid #ccc;
        border-top: none;
        padding: 10px;
        border-radius: 0 5px 5px 5px;
    }
```

You can see it in action at https://mapsui.com/samples

**Comment by VengaYBanzai** [comment]
Thank you very mutch. I actually tried it but did not work in the first place. Nevetheless after settings everything up again it works.

**Comment by pilotshamn** [comment]
Hi VengaYBanzai
I have the same  problem i cant change the size of the MapControlComponent however i try (create a new project)
How did you solve this issue?
Thank you
